.vscode/tools/setup_vscode.py

```diff
@@ -91,7 +100,7 @@ def overwrite_python_analysis_extra_paths(isaaclab_settings: str) -> str:
     # deal with the path separator being different on Windows and Unix
     path_names = path_names.replace("/", os.sep)
```

Comment:
It seems like regex parses "\" in the paths on windows as escapes and will error out since they aren't valid characters to escape. we may need to escape the backslashes in the paths or just use forward slashes.